ftplugin/javascript/cfi.vim

```diff
@@ -18,10 +18,31 @@ set cpo&vim
 " Options
 " ============================================================================
 
-" whether or not to show type of function assignment, e.g. 'named expression'
-" or 'anon expression' or 'anon' or 'arrow'
-let g:cfi_javascript_show_assignment =
-      \ get(g:, 'cfi_javascript_show_assignment', 0)
+" Configure display verbosity
+" By default, show everything
+let s:show_defaults = {
+      \   'assignment': 1,
+      \   'variable_name': 1,
+      \   'function_type': 1,
+      \   'function_name': 1,
+      \   'function_arguments': 1,
+      \   'function_body': 1,
+      \ }
+if exists('g:cfi_javascript_show')
+  " Add any missing values, preserve the user settings
+  call extend(g:cfi_javascript_show, s:show_defaults, 'keep')
+else
+  let g:cfi_javascript_show = s:show_defaults
+endif
+
+" Deprecation warnings for the old style config
+" Remove it after a while (couple months, at least
+" "Now": 2016-03-13
+for var in ['g:cfi_javascript_show_assignment', 'g:cfi_javascript_show_only_fname']
```

Comment:
Yeah, this was kinda dumb.
Chalk it up to a reminder to change my own settings.